cmd/manager-hub/main.go

```diff
@@ -87,21 +87,22 @@ func main() {
 
 	setupLogger.Info("Creating manager", "git commit", commit)
 
-	options := ctrl.Options{Scheme: scheme}
-
-	options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile))
+	cfg, err := config.ParseFile(configFile)
```

Comment:
had the new controller-runtime version removed support for parsing option? If not, then why not use its code?

Reply:
It has deprecated its config parser: [issue](https://github.com/kubernetes-sigs/controller-runtime/issues/895), [Go doc](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.15.0/pkg/config#ControllerManagerConfiguration)
Users are asked to move to their own config management system, this is what this PR does.